app/src/main/java/org/mozilla/fenix/home/recentvisits/view/RecentlyVisited.kt

```diff
@@ -315,7 +315,7 @@ private fun RecentlyVisitedMenu(
         onDismissRequest = { onDismissRequest() },
         modifier = Modifier
             .background(color = FirefoxTheme.colors.layer2)
-            .height(52.dp)
+            .height(64.dp) // minimum height of a menu with one item
```

Comment:
I recall from talking @eliserichards that this 52dp is intentional. I can't find the commit history of this change at the moment, but I think we want to double check this.

But as aside, I am also wondering why our refactor of RecentlyVisited from #22534 did not preserve the commit history for this file. 😦

Reply:
52 dp might've been intentional but it's according to the compose specs.
64 is the absolute minimum based on how the DropdownMenu works.

Reply:
The history continues to be available in git, just need to jump through more commits.
Found the ticket/PR in which this was implemented - https://github.com/mozilla-mobile/fenix/pull/21298

Reply:
Ah yes, this is the PR I have attempted to find, but could not actually find it for the life of me.

I am wondering now if we even need to specify a given `height` if it's already the default in the compose implementation.

Reply:
I'd be fine either way.
Though since this was a bug and an interesting discussion I'd say having this height value and comment here, just for brevity and future reference would be nice.

Reply:
```suggestion
            .height(64.dp) // The absolute minimum height of a menu with one item #22508
```
Just one suggested change for the comment if we aren't removing the `height`. Keeping this does make me a bit nervous for any Compose version bump in the future since these defaults could change, and we end up with some unexpected behaviour because of the hardcoded height. The alternative would be to add a note in the kDoc.

Reply:
Good point. Removed the hardcoded height.
Tested the it works as expected.